packages/features/bookings/lib/getBookingResponsesSchema.ts

```diff
@@ -122,6 +122,12 @@ function preprocess<T extends z.ZodType>({
           ? z.string()
           : z.string().refine(async (val) => {
               const { isValidPhoneNumber } = await import("libphonenumber-js");
+
+              // Allow Australian numbers starting with +61493 or +610493
+              if (val.startsWith("+61493") || val.startsWith("+610493")) {
+              return true;
+              }
```

Comment:
I would really not want this to be hardcoded here. That's the job of libphonenumber-js. If it's not fixed there, maybe we should create a new module isValidPhoneNumber that would abstract it out till it's fixed in libphonenumber-js.

Also, this change would make 61493 a valid phone number which would be wrong as there has to be more digits in the end.

Reply:
@hariombalhara yeah this PR https://github.com/calcom/cal.com/pull/12519 is better

Reply:
someone mentioned that the current implementation is working perfectly so that's why i stop working on it. if #12519 solved the issue that is great :)